lib/tasks/service_providers.rake

```diff
@@ -23,4 +23,9 @@ namespace :service_providers do
     issuers = ENV.fetch('ISSUERS', '').split(',')
     DestroyUnusedProviders.new(issuers).run
   end
+
+  desc 'Sync service provider configurations'
+  task sync: :environment do
+    ServiceProviderSeeder.new.run
+  end
```

Comment:
Anything I should consider around this?

Reply:
looks good! You're on your way to a career maintaining 10+ year old Rails apps!